src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -940,6 +941,26 @@ public string[] Split(string input, int count, int startat)
             return RegexReplacement.Split(this, input, count, startat);
         }
 
+#if FEATURE_COMPILED
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "assemblyname", Justification = "Microsoft: already shipped since v1 - can't fix without causing a breaking change")]
+        public static void CompileToAssembly(RegexCompilationInfo[] regexinfos, AssemblyName assemblyname)
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
Mind adding a string like  
 7869:     <value>This platform does not support server side WebSockets.</value>